--- a/sqlite-amalgamation-3240000/sqlite3.c
+++ b/sqlite-amalgamation-3240000/sqlite3.c
@@ -948,9 +948,9 @@ SQLITE_PRIVATE const char **sqlite3CompileOptions(int *pnOpt){
 */
 #ifndef SQLITE_DISABLE_LFS
 # define _LARGE_FILE       1
-# ifndef _FILE_OFFSET_BITS
-#   define _FILE_OFFSET_BITS 64
-# endif
+// # ifndef _FILE_OFFSET_BITS
+// #   define _FILE_OFFSET_BITS 64
+// # endif
 # define _LARGEFILE_SOURCE 1
 #endif

--
2.18.0
